spec/cc/engine/config_upgrader_spec.rb

```diff
@@ -17,6 +17,12 @@ def get_true
           Enabled: false
         Style/FrozenStringLiteralComment:
           Enabled: false
+        Style/HashEachMethods:
+          Enabled: false
+        Style/HashTransformKeys:
+          Enabled: false
+        Style/HashTransformValues:
+          Enabled: false
```

Comment:
While this change does cause the spec to pass, I'm not sure it's right way to handle the stdout warning about new cops.

Reply:
I would say let's turn them ON!

Reply:
Your wish is my command! 51b5aa4

Reply:
Makes sense to enable them since they are enabled by default in Rubocop.

Reply:
Note that they are not enabled by default in RuboCop, but rather "pending" which means the users have to decide whether to enable them or not. We'll likely enable them by default in version 1.0.

Reply:
Ok, lets leave it on in the spec, but keeping it out of the default config, so it warns users to make a decision, just like rubocop itself.

Reply:
Ok, lets leave it on in the spec, but keeping it out of the default config, so it warns users to make a decision, just like rubocop itself.

Reply:
Done in a81a64e. ✅

Reply:
Github is not helping much today... seems to be having issues.
I've tried to merge this early today and seems like the branch got into a weird state instead.

... just checked, it actually merged the changes in, but kept the PR open! So weird.

Reply:
The diff is now weird, but your changes are there.
I will be releasing this soon.